oak-parent/pom.xml

```diff
@@ -69,8 +68,11 @@
     <testcontainers.version>1.17.5</testcontainers.version>
     <pax-exam.version>4.13.1</pax-exam.version>
     <groovy.version>2.5.20</groovy.version>
-    <java.version>1.8</java.version>
-    <java.version.signature>java18</java.version.signature>
+
+    <java.version>11</java.version>
+    <maven.compiler.release>${java.version}</maven.compiler.release>
+    <maven.compiler.source>${java.version}</maven.compiler.source>
```

Comment:
redundant (set in asf parent)

Reply:
which part exactly is redundant?

Reply:
https://github.com/apache/maven-apache-parent/blob/5faa5c33e3035e5c83af1ed40b4abf6b131a1149/pom.xml#L94

Reply:
Just remove line 74